CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 ## [0.6.1] - TBD
 - Bugfix: Fix runtime ignoring `Formula.key` for the root formula.
 - [formula-android] Adding main thread check before notifying fragments. 
+- Migrated `formula-android` to use core `Formula` interface
```

Comment:
Does this have any impact on end consumers?

Reply:
It will now detect duplicate contract registrations explicitly and crash.